test-end-to-end/android-maestro/e2e-tests/001_004_admin_can_move_content.yaml

```diff
@@ -0,0 +1,35 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/001_004_admin_can_move_content"
+  TESTNAME: "001_004_admin_can_move_content"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+- runFlow:
+    file: "../common/add_content.yaml"
+    env:
+      TESTFILENAME: Epub_Content1.epub
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+- tapOn: "SAVE"
+# Add new folder to move content
+- tapOn:
+    id: "floating_action_button"
+- tapOn: "New folder"
+- tapOn: "Title*"
+- inputText: "Test Folder"
+- tapOn: "SAVE"
+- takeScreenshot: "${TESTRESULTSDIR}/screenshots/before_moving_content_to_new_folder"
+- longPressOn:
+    text: "E-book"
+- tapOn: "Move"
+- tapOn: "Folder"
+- tapOn: "Move entries to this folder"
+- takeScreenshot: "${TESTRESULTSDIR}/screenshots/after_moving_content_to_new_folder_test"
+- tapOn: "Folder"
+- assertVisible: "E-book"
```

Comment:
Here; again, we should have an assertion that the title has changed to make sure that we are in the expected folder. (e.g. after line 34 tapOn folder.

Reply:
This was added:

assertVisible: "Test Folder"

But this won't have the intended effect. "Test Folder" needs to be in the title. The assert visible needs to include both text and the id of the title (app_title). 

This can pass if the folder hasn't opened yet, because "Test Folder" will still be in the list.